source/speech/speech.py

```diff
@@ -1125,8 +1125,15 @@ def speakTextInfo(
 	)
 
 	speechGen = GeneratorWithReturn(speechGen)
+	symbolLevel: Optional[characterProcessing.SYMLVL] = None

```

Comment:
I'm hesitant to make this change as `characterProcessing.SYMLVL.NONE` == 0 and isn't the same as `None` from what I can tell. There are cases where we do `if symbolLevel is None`.

Reply:
I suppose it could be option to represent "not set" yet (eg in a GUI). Is this actually possible though?

Otherwise, shouldn't there always be some value?

Reply:
It's an unset value when determining the level of speech to report, that is processed later in `speak`. It's not driven by a UI, it's contextual in the processing.

Reply:
Is there a default level? Presumably it would be an error to come out the other end of the processing as `None`

Reply:
Yes, it is later set to `config.conf["speech"]["symbolLevel"]` if it is not otherwise set earlier

Reply:
So from the docs of `def speak`:
>@param symbolLevel: The symbol verbosity level; C{None} (default) to use the user's configuration.